src/plugins/data/public/antlr/shared/utils.ts

```diff
@@ -93,41 +96,82 @@ export const fetchData = (
   });
 };
 
-// Specific fetch function for table schemas
-// TODO: remove this after using data set table schema fetcher
-export const fetchTableSchemas = (tables: string[], api: any, queryString: QueryStringContract) => {
-  return fetchData(
-    tables,
-    (table, dataSourceId, title) => ({
-      query: { query: `DESCRIBE TABLES LIKE ${table}`, format: 'jdbc' },
-      df: {
-        meta: {
-          queryConfig: {
-            dataSourceId: dataSourceId || undefined,
-            title: title || undefined,
-          },
+export const fetchColumnValues = async (
+  table: string,
+  column: string,
+  services: IDataPluginServices,
+  fieldInOsd: IndexPatternField | undefined
+): Promise<any[]> => {
```

Comment:
I originally wrote this to return a string array, but I changed it to `any` because if the value suggestions change in the future and return more types than just string, this method would be able to handle those cases too. 
Should I change it back to `string[]`?